### ISSUE TITLE
 can not remove the layer

### ISSUE THREAD

**Issue by lannio** [cga-harvard/geonode#326]
When I delete the layer, the layer still exists. 
The delete layer is invalid and there are no errors on the console.
![11](https://user-images.githubusercontent.com/33817803/38255312-187e2af8-378e-11e8-81c1-95ea42e7e456.jpg)
![12](https://user-images.githubusercontent.com/33817803/38255343-30053b26-378e-11e8-94ae-3fa0e99aab53.jpg)
When I delete a layer in the administrator interface, there is the following issue.
![14](https://user-images.githubusercontent.com/33817803/38255663-faedaf44-378e-11e8-9aff-3dddce68a4fd.jpg)
![13](https://user-images.githubusercontent.com/33817803/38255592-c793abbc-378e-11e8-8335-1e5eedba264c.jpg)

**Comment by capooti** [comment]
@lannio unfortunately I cannot replicate this problem on my environment and in the staging, so it could be related to the configuration. I would suggest to post this error to the GeoNode user lists, which is the recommended way to ask help on stuff related to GeoNode.
Let me know if you want to do it on your own, or if you prefer me to do it for you.

**Comment by capooti** [comment]
here is the list (you need to register for it): https://lists.osgeo.org/cgi-bin/mailman/listinfo/geonode-users

**Comment by capooti** [comment]
@lannio they suggested me that the error is caused by the new GoeWebCache REST context.

Do you have this lines in your configuration?

https://github.com/GeoNode/geoserver-geonode-ext/blob/master/geoserver/data/geonode_v212x/geofence/geofence-server.properties#L23

https://github.com/GeoNode/geoserver-geonode-ext/blob/master/geoserver/data/geonode_v212x/monitoring/db.properties#L24

In case you haven't them you need to update GeoServer. Backup your data directory, remove the camp/downloaded and camp/geoserver directories (even better rename them), stop GeoServer and run again paver setup. Then copy the data directory in the new GeoServer instance and restart GeoServer.

**Comment by lannio** [comment]
Thank you for your reply.
The two files you mentioned above I found appear in two places, one in camp/geoserver/data, and one in camp/geoserver/geoserver/data.
So what file are you referring to?
![21](https://user-images.githubusercontent.com/33817803/38283887-271a219a-37eb-11e8-8ba4-1ab3ab7e74e2.jpg)
![22](https://user-images.githubusercontent.com/33817803/38283888-274f5bbc-37eb-11e8-87ec-dee7b1f44886.jpg)
I found that in camp/geoserver/geoserver/data, the content of the files are the same as the content of the link.
And in camp/geoserver/data, the content of the db.properties is the same as the content of the link, but geofence-server.properties is empty.
Also I found that in camp/geoserver/data/geofence.old, the file of geofence-server.properties is the same as the content of the link.

In short, camp/**geoserver**/data/**geofence.old**/geofence-server.properties,  camp/**geoserver**/data/**monitoring**/db.properties, camp/**geoserver/geoserver**/data/**geofence**/geofence-server.properties and camp/**geoserver/geoserver**/data/**monitoring**/db.properties are the same as the link, but camp/**geoserver**/data/**geofence**/geofence-server.properties is empty.
Should I fill in the corresponding content in camp/**geoserver**/data/**geofence**/geofence-server.properties?
In addition, which folder we need to back up,  camp/**geoserver**/data or camp/**geoserver/geoserver**/data?

**Comment by capooti** [comment]
Hi @lannio for some reason I didn't notice this message, sorry for being late with it.
geoserver/geoserver/data are just source files which are copied to geoserver/data when running paver setup. So you should change whatever is in geoserver/data, and don't look at all in geoserver/geoserver/data.
Same applies to backups: your geoserver data directory is camp/geoserver/data (if you check in camp/geoserver/data/workspace you should find the configuration of all of the layers which are in your GeoServer instance).
Hope it is clear.

**Comment by lannio** [comment]
Ok, I get it.
I fill in the corresponding content in camp/geoserver/data/geofence/geofence-server.properties ,and I can remove layers.
